fuel/src/test/kotlin/com/github/kittinunf/fuel/EncodingTest.kt

```diff
@@ -21,6 +21,18 @@ class EncodingTest : BaseTestCase() {
         assertThat(request.url.toString(), isEqualTo("http://www.example.com/test?a=b"))
     }
 
+    @Test
+    fun testEncodingWithBaseUrlStringWithLastSlash() {
+        val request = Encoding(
+                httpMethod = Method.GET,
+                baseUrlString = "http://www.example.com/",
+                urlString = "test",
+                parameters = listOf("a" to "b")
+        ).request
+
+        assertThat(request.url.toString(), isEqualTo("http://www.example.com/test?a=b"))
+    }
+
```

Comment:
Should we add a test for the problem that you face, and see whether the code fixes or not?

```
baseUrlString = "http://www.example.com/"
urlString = "/test"
```

Reply:
Yes, please.

Reply:
Could you please add them? 😄

Reply:
@kindraywind 
Sure.
There was a mistake in adding test.